core/src/main/java/cz/xtf/core/config/OpenShiftConfig.java

```diff
@@ -6,6 +6,9 @@
     public static final String OPENSHIFT_VERSION = "xtf.openshift.version";
     public static final String OPENSHIFT_NAMESPACE = "xtf.openshift.namespace";
     public static final String OPENSHIFT_BINARY_PATH = "xtf.openshift.binary.path";
+    public static final String OPENSHIFT_BINARY_CACHE_ENABLED = "xtf.openshift.binary.cache.enabled";
+    public static final String OPENSHIFT_BINARY_CACHE_PATH = "xtf.openshift.binary.cache.path";
+    public static final String OPENSHIFT_BINARY_CACHE_PATH_DEFAULT_VALUE = "/tmp/xtf/oc-cache";
```

Comment:
I think there could be used more OS agnostic approach and store `oc` in folder like: 
```Files.createTempDirectory(...);```

Reply:
sure good idea, thanks @mnovak1 I will apply this change

Reply:
Wouldn't it effectively disable caching with default value? A new temporary directory would be created with each execution, no?

Reply:
@simkam I update the PR, I cannot create a new temp directory every time, I just used java.io.tmpdir property to retrieve the temp root dir, and I used Paths.get(..) to concatenate root temp and folder